services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> QuerySet:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,  # movie=movie_id
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: datetime = None) -> QuerySet:
+    if session_date:
+        formated = datetime.strptime(session_date, "%Y-%m-%d").date()
+        return MovieSession.objects.filter(show_time__date=formated)
+        # return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> QuerySet:
+    return MovieSession.objects.get(id=movie_session_id)  # id
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: datetime = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> QuerySet:
```

Comment:
The return type of `update_movie_session` should be `MovieSession` instead of `QuerySet`, as the function updates and returns an instance of the model.